packages/renderer/src/lib/objects/KubernetesObjectUI.ts

```diff
@@ -0,0 +1,24 @@
+/**********************************************************************
+ * Copyright (C) 2025 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+
+export interface KubernetesObjectUI {
+  name: string;
+  namespace?: string;
+  kind?: string;
```

Comment:
Should we take this opportunity to set this property to mandatory, all resources have a kind 

> All resource types have a concrete representation (their object schema) which is called a kind


From https://kubernetes.io/docs/reference/using-api/api-concepts/